### PR TITLE
update Safari TP to release 9

### PR DESCRIPTION
**Pull request by chicoxyzzy**
https://developer.apple.com/safari/technology-preview/release-notes/

### CODE REVIEW

File: data-es2016plus.js [hunk 1]
```diff
@@ -175,6 +175,7 @@ exports.tests = [
           chrome47:        true,
           edge14:          true,
           firefox43:       true,
+          safaritp:        true,
```
Comment (zloirock) [inline data-es2016plus.js]: Duplicate key.

Comment (chicoxyzzy) [inline data-es2016plus.js]: yes it was mistakenly added. removed this one. thank you

Comment (chicoxyzzy) [inline data-es2016plus.js]: @zloirock noticed you've point to wrong line. Mistakenly added twice was `Array.prototype.includes` feature. This one (`%TypedArray%.prototype.includes`) was added in TP 9 so this one is not duplicate

**Comment by mkurz** [comment]
Why is there a "SF 10" which reaches "100%" and "SF TP" which reaches "only" "99%"? Isn't "SF TP" = "SF 10" - because SF 10 hasn't been released yet?

**Comment by chicoxyzzy** [comment]
@mkurz SF TP is not SF 10. Both are unstable versions

File: data-es2016plus.js (hunk 1 above)
Comment (zloirock) [inline data-es2016plus.js]: [`jshint` не устроила эта строчка :)](https://travis-ci.org/kangax/compat-table/builds/146200034)

**Comment by chicoxyzzy** [comment]
btw looks like TP got `Symbol.isConcatSpreadable` support so I should update es6 table

**Comment by mkurz** [comment]
@chicoxyzzy So what's the difference between the two? I assume SF10 will end up as the final Safari 10 release. But what about SF TP?

File: data-es2016plus.js (hunk 1 above)
Comment (zloirock) [inline data-es2016plus.js]: > 236

Hmm... I added this comment to 178 and I checked it. Looks strange.

**Comment by chicoxyzzy** [comment]
@mkurz Safari 10 is shipping with macOS Sierra developer and public betas. Safari TP is available under both macOS Sierra and OS X El Capitan and it is kind of nightlies versions of Safari which are based on WebKit Nightlies (which are available for OS X El Capitan only).
So usually new SF 10 build is based on latest SF TP which is based on some of recent versions of WK Nightly. WK updates more frequently then SF TP which updates more frequently than SF 10

**Comment by mkurz** [comment]
Thank you for the explanation. I was confused because SF 10 scored 100% but SF TP "only" 99% (before this pull request). I would have expected it the other way round - but probably in the TP "nightly" a feature got removed or disabled which threw it back to 99%.

**Comment by chicoxyzzy** [comment]
@mkurz previous TP was released earlier than SF 10. Todays TP 9 got `Symbol.isConcatSpreadable` support. I didn't check es6 tab because I mistakenly thought all SF browsers already got 100% es6 support .Thank you for pointing out on that! :)

**Comment by mkurz** [comment]
@chicoxyzzy This makes totally sense of course :smile:

**Pull request merged**